### DISCUSSION TITLE
Cannot using responsive utility

### DISCUSSION THREAD

**Discussion by manusiakemos** [tailwindlabs/tailwindcss#12465] [Help]
**version of Tailwind CSS** : 3.3.5

**build tool** : laravel  10 + vite

**version of Node.js?** : v16.16.0

**browser** : Chrome

**operating system**: Mac OS

**Reproduction URL**

my problem is when i using responsive class it only process the first class.

for example if i using w-0 md:w-32. on desktop width still 0.

![image](https://github.com/tailwindlabs/tailwindcss/assets/31971611/20d03ab9-74c9-4ca5-9d0f-6954a2461cdc)

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 

Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.
